hwstats/models.py

```diff
@@ -15,7 +15,7 @@ class SysProcess(Base):
 
     __tablename__ = "system_process"
 
-    # id: Mapped[int] = mapped_column(primary_key=True)
+    id: Mapped[int] = mapped_column(primary_key=True)
```

Comment:
What was your reasoning for adding this back in? I didn't think we could have two primary keys, and we need the pidHash to be the primary key in order to maintain unique processes in the table.

Reply:
I thought that I commented on the line, but I hadn't.